update_rootfs.sh

```diff
@@ -39,15 +39,14 @@ else
 	printf "\t\033[1m- Mounted RootFS:\033[0m '%s' at '%s'\n" "$ROOTFS" "$MOUNT_POINT"
 fi
 
-printf "\t\033[1m- Calculating Random Checksum\033[0m\n"
-IMG_CS=$(dd if=/dev/urandom bs=1 count=8 2>/dev/null | md5sum | cut -c1-8)
-printf "\t\033[1m- Using Checksum:\033[0m %s\n" "$IMG_CS"
+BUILD_ID=$(git -C "$HOME/$REPO_DIR/internal" rev-parse --short HEAD)
```

Comment:
Might as well add `REPO_INTERNAL="${REPO_INTERNAL:-internal}"` to this script too and reference it.  The only other thing would be perhaps to add a init script will all these values so it can be called and easily changed/added to in the future.

Reply:
Done! (Didn't factor into its own script yet mostly because it's late, but am happy to do that in the future. :D)